### PR TITLE
Modified library/port_alias.py to have flexibility to include internal interface

### PR DESCRIPTION
**Pull request by oxygen980**
### Description of PR
Modified library/port_alias.py to have flexibility to include=True or False the internal interfaces in VOQ system.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
To have flexibility to include or exclude internal interfaces inside a testcase.

#### How did you do it?
Added 'include_internal' flag in port_alias.py
#### How did you verify/test it?
Tested on a VOQ chassis

### CODE REVIEW

**Comment by shubav** [comment]
@wangxin could you kindly review this? Thanks

File: ansible/library/port_alias.py [hunk 1]
```diff
@@ -182,10 +183,15 @@ def main():
                 num_asic = multi_asic.get_num_asics()
             except Exception, e:
                 num_asic = 1
+        # if m_args['include_internal'] is not None:
+        #     include_internal = m_args['include_internal']
```
Comment (yxieca) [inline ansible/library/port_alias.py:187]: These looks like debug code that should be removed now?

Comment (oxygen980) [reply ansible/library/port_alias.py]: Resolved

**Comment by shubav** [comment]
Does it look ok, @yxieca ? Can it be merged or does it need more reviewers? Thanks.

**Review by yxieca — APPROVED** [review]

**Pull request merged**